### PR TITLE
add cwe field to vulnerability message in v1beta1

### PR DESCRIPTION
**Pull request by hkadakia**
This PR addresses changes as discussed in https://github.com/grafeas/grafeas/issues/519.

### CODE REVIEW

File: proto/v1beta1/vulnerability.proto [hunk 1]
```diff
@@ -149,7 +149,11 @@ message Vulnerability {
   // The full description of the CVSS for version 2.
   CVSS cvss_v2 = 7;
 
-  // Next free ID is 8.
+  // A list of CWE for this vulnerability.
+  // For details, see: https://cwe.mitre.org/index.html
+  repeated string cwe = 8;
```
Comment (aysylu) [inline proto/v1beta1/vulnerability.proto:154]: do we expect the same CWEs to apply on a per-distro basis? or will this vary?

Comment (hkadakia) [reply proto/v1beta1/vulnerability.proto:154]: same CWEs should be for all distros.

Comment (hkadakia) [reply proto/v1beta1/vulnerability.proto:154]: On checking further, CWEs are just common weakness types, they are general. CVEs are specific vulnerabilities that can affect one or more distros. We could keep CWE as a general field in the vulnerability message as in the PR.

Comment (aysylu) [reply proto/v1beta1/vulnerability.proto:154]: Got it, thanks for checking! SGTM. I checked a couple CVEs as well and it appears the CWE association is the same across distros.

**Review by aysylu — APPROVED** [review]

**Comment by aysylu** [comment]
Added the co-maintainer, Wiktor, for another pair of eyes. LGTM on my side.

**Review by wkozlik — APPROVED** [review]
LGTM

**Pull request merged**